tasks/4_stand-in-line/stand-in-line.js

```diff
@@ -0,0 +1,4 @@
+export function nextInLine(niz, broj) {
```

Comment:
As we work in an international company, English is the official language that we use here, so please use it in future.

Reply:
Changed.